test/functional/lua/fs_spec.lua

```diff
@@ -166,8 +179,8 @@ describe('vim.fs', function()
       io.open('testd/a/b/c/b4', 'w'):close()
       io.open('testd/a/b/c/c4', 'w'):close()
 
-      local function run(dir, depth, skip)
-        return exec_lua(function()
+      local function run(dir, depth, skip, follow)
+        return exec_lua(function(follow_sym)
```

Comment:
nit: usual convention is something like:
```suggestion
        return exec_lua(function(follow_)
```